### DISCUSSION TITLE
How can I change the font size of the blog name and its description?

### DISCUSSION THREAD

**Discussion by 5hEn918** [alshedivat/al-folio#2263] [Q&A] [answered]
Currently I can set up the blog name and description in the "_config.yml" file, but I wonder how to change the font size of it?
![image](https://github.com/alshedivat/al-folio/assets/161289944/1c73982a-7e5b-49bf-9b64-c5ca8042aa74)

**Comment by george-gca** [comment, marked as answer]
You will basically need to play with the css of the page. I'd recommend using the developer tools available in [Chrome](https://developer.chrome.com/docs/devtools/overview/) or [Firefox](https://firefox-source-docs.mozilla.org/devtools-user/) to test the changes first, then apply the changes in the respective class in `_sass/_base.scss`.

**Reply by 5hEn918** [reply]
Thanks for your advice!
